src/geo/map/cartodb-layer.js

```diff
@@ -120,6 +120,15 @@ var CartoDBLayer = LayerModelBase.extend({
     var meta = this.get('meta');
     var stats = meta && meta.stats;
     return stats && stats.estimatedFeatureCount;
+  },
+
+  getSourceId: function () {
+    var source = this.getSource();
+    return source && source.id;
+  },
+
+  getSource: function () {
+    return this.get('source');
```

Comment:
Something to think about here is wether we want to keep using a Backbone attribute to store the the "source" or we should go with an instance variable (eg: `this._source`)... Good thing about Backbone attributes is that:
1. They can be changed "in bulk".
2. Changing an attribute will trigger a `change` event.

On the other hand, if we go with an instance variable, we'll need to implement a setter (`setSource`) and we'll have to be more explicit about side-effects (eg: reloading the visualisation).